c/array-fpi/Makefile

```diff
@@ -0,0 +1,6 @@
+LEVEL := ../
+
+CLANG_WARNINGS := \
+	-Wno-error=uninitialized \
```

Comment:
This suppresses warnings about uninitialized variables. Is this necessary? Are the programs still safe? (Reading from uninitialized variables is undefined behavior and must not exist in the program.)

Reply:
@PhilippWendler Thanks for pointing this out. The suppression of the warnings is not necessary. I will remove the CLANG_WARNINGS variable from the make.

Uninitialized variables were considered to have an arbitrary (non-deterministic) value as BMC would. Thus, programs are still safe even when uninitialized variables are present. I now see that this might be an issue for some techniques. I will try to remove uninitialized variables and commit the changes. Thanks.

Reply:
Yes, you need to initialize all variables before reading from them, typically with calls to `__VERIFIER_nondet_int` etc.

Reply:
I have gone through every program and none of them currently have a uninitialized read on a variable or an array element. In case you find one in the review, please let me know. Otherwise, we can resolve this.